pack/fhv/fhvc.json

```diff
@@ -4020,7 +4020,7 @@
     "type_code": "enemy"
   },
   {
-    "code": "10693",
+    "code": "10693a",
```

Comment:
We can't change the code of these cards anymore due to apps like ArkhamCards campaign guide depending on it.